tests/unit/models/360-core.js

```diff
@@ -232,6 +230,19 @@ describe("SerSol 360 Core Model >", function() {
       var target = serSol360Core.getTarget(title);
       expect(target).toBeUndefined();
     });
+
+    it("should retrieve target element when the results title element contains the journal title", function() {
+      var title = titles[0];
+      var journalName = $(title.target).find(".results-title").text();
+
+      $("div[ui-view='searchResults']").find(".results-title:contains('" + journalName + "')")[0].innerHTML += " Terms of Use";
+
+      var target = serSol360Core.getTarget(title);
+
+      expect(target).toBeDefined();
+      expect(target.innerHTML).toContain("The New England journal of medicine");
+      expect(target.innerHTML).toContain("The New England journal of medicine Terms of Use");
```

Comment:
Good call!

Should we add a test for checking unexpected case changes, too?  I see the code does a `lowercase` on the title strings it compares, but I don't think we explicitly test for that, correct?

Reply:
Definitely. Added: https://github.com/thirdiron/browzine-discovery-service-adapters/pull/37/files#diff-bb9050811ef596c1707abe110a18bd52R249

Thanks!